pkl-cli/src/test/kotlin/org/pkl/cli/CliTestRunnerTest.kt

```diff
@@ -429,18 +429,55 @@ class CliTestRunnerTest {
     assertThat(out.toString().stripFileAndLines(tempDir))
       .isEqualToNormalizingNewlines(
         """
-      module test
-        examples
-          ❌ nums
-            
-             Output mismatch: Expected "nums" to contain 1 examples, but found 2
-      ❌ 0.0% tests pass [1/1 failed], 0.0% asserts pass [1/1 failed]
+        module test
+          examples
+            ❌ nums
+              
```

Comment:
nit: maybe the extra whitespace / line could be removed too? :-? Or maybe it's an artifact of the tests, when removing the location string? I'm trying to find the exact place where the additional `\n` is added but I see a `location` that's not showing in here :-?